.gitlab-ci.yml

```diff
@@ -150,7 +69,7 @@ Deploy nodes to prod:
     # +----------------------------+
     # |  🔴 Deploy SSV Mainnet nodes  |
     # +----------------------------+
-    # - .k8/production/mainnet/scripts/deploy-cluster-1-4.sh $DOCKER_REPO_INFRA_PROD $CI_COMMIT_SHA ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
+    - .k8/production/mainnet/scripts/deploy-cluster-1-4.sh $DOCKER_REPO_INFRA_PROD $CI_COMMIT_SHA ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
```

Comment:
not sure we want to leave mainnet deployment by default
@moshe-blox wdyt?

Reply:
@y0sher @systemblox yeah we want to avoid any mainnet deployments by default (like today)

Reply:
@y0sher @moshe-blox ok. I will update the GitLab-ci file to deploy one operator, and then we can update the remaining nodes one by one.